frontend/src/scenes/billing/BillingLimit.tsx

```diff
@@ -0,0 +1,118 @@
+import { LemonButton, LemonInput } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { Field, Form } from 'kea-forms'
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
+import { useRef } from 'react'
+
+import { BillingProductV2Type } from '~/types'
+
+import { billingLogic } from './billingLogic'
+import { billingProductLogic } from './billingProductLogic'
+
+export const BillingLimit = ({ product }: { product: BillingProductV2Type }): JSX.Element | null => {
+    const limitInputRef = useRef<HTMLInputElement | null>(null)
+    const { billing, billingLoading } = useValues(billingLogic)
+    const { isEditingBillingLimit, customLimitUsd } = useValues(
+        billingProductLogic({ product, billingLimitInputRef: limitInputRef })
+    )
+    const { setIsEditingBillingLimit, setBillingLimitInput, submitBillingLimitInput } = useActions(
+        billingProductLogic({ product })
+    )
+
+    if (billing?.billing_period?.interval !== 'month' || !product.subscribed) {
+        return null
+    }
+
+    return (
+        <Form formKey="billingLimitInput" props={{ product: product }} logic={billingProductLogic} enableFormOnSubmit>
+            <div className="border-t border-border p-8" data-attr={`billing-limit-input-${product.type}`}>
+                <h3 className="mb-2">Billing limit</h3>
+                <div className="flex">
+                    {!isEditingBillingLimit ? (
+                        <div className="flex items-center justify-center gap-1">
+                            {customLimitUsd ? (
+                                <>
+                                    <Tooltip title="Set a billing limit to control your recurring costs. Some features may stop working if your usage exceeds your limit.">
+                                        <span>
+                                            You have a <b>${customLimitUsd}</b> billing limit set for {product?.name}
```

Comment:
```suggestion
                                            You have a <b>${customLimitUsd}</b> billing limit set for {product?.name}.
```

Reply:
Can we also lowercase product.name so it's `product analytics` instead of `Product analytics`?